.docker/web/clean.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+echo "===== CLEANING COMPILED FILES ====="
+rm -rf /home/airline/airline/airline-web/target
+rm -rf /home/airline/airline/airline-web/project/target
+rm -rf /home/airline/airline/airline-web/project/project
```

Comment:
it seems wrong to me that a /target folder is stored both in /project/target and /project/project, is that a mishap in the activator or just how scala works?